samples/nrf_desktop/configuration/nrf52_pca20037/led_state_def.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include "led_state.h"
+
+/* This file must be included only once */
+const struct {} led_state_def_include_once;
+
+/* Map function to LED ID */
+static const u8_t led_map[LED_ID_COUNT] = {
+	[LED_ID_SYSTEM_STATE] = 0,
+	[LED_ID_PEER_STATE] = 1
```

Comment:
I ignored this file for now as FW configuration does not match HW. This will have to be fixed under separate PR. We also need to discuss how to show various things with only 1 led.